ui/public/locales/pt_BR.json

```diff
@@ -2061,246 +2727,631 @@
+"message.setting.updated": "Configura\u00e7\u00e3o atualizada:",
+"message.setup.physical.network.during.zone.creation": "Ao adicionar uma zona avan\u00e7ada, voc\u00ea precisa configurar uma ou mais redes f\u00edsicas. Cada rede corresponde  \u00e0 uma interface de rede no virtualizador. Cada rede f\u00edsica pode ser utilizada para transportar um ou mais tipos de tr\u00e1fego, com certas restri\u00e7\u00f5es sobre como eles podem ser combinados. <br/> <strong> Arraste e solte um ou mais tipos de tr\u00e1fego </ strong> em cada rede f\u00edsica.",
+"message.setup.physical.network.during.zone.creation.basic": "Quando adicionar uma zona b\u00e1sica, voc\u00ea pode configurar uma rede f\u00edsica, que corresponde a uma interface de rede no virtualizador. A rede carrega diversos tipos de tr\u00e1fego.<br/><br/>Voc\u00ea pode <strong>adicionar e remover</strong> outros tipos de tr\u00e1fego na mesma interface de rede f\u00edsica.",
 "message.setup.successful": "Cloud configurada com sucesso!",
+"message.specify.tag.key": "Por favor, especifique uma chave de etiqueta",
 "message.specify.tag.key.value": "Por favor especifique chave e valor da tag",
+"message.specify.tag.value": "Por favor, especifique um valor para a etiqueta",
 "message.specify.url": "Por favor especifique a URL",
-"message.step.1.desc": "Por favor, selecione um template para a sua nova inst\u00e2ncia virtual. Voc\u00ea pode tamb\u00e9m escolher um template limpo e instalar a partir de uma imagem ISO.",
+"message.step.1.desc": "Por favor, selecione um template para a sua nova Inst\u00e2ncia virtual. Voc\u00ea pode tamb\u00e9m escolher um template limpo e instalar a partir de uma imagem ISO.",
 "message.step.2.continue": "Selecione o plano",
-"message.step.3.continue": "Seleciona a oferta de disco",
+"message.step.3.continue": "Seleciona a Oferta de Disco",
 "message.step.4.continue": "Selecione pelo menos uma rede para continuar",
-"message.step.4.desc": "Selecione a rede principal que a sua inst\u00e2ncia virtual estar\u00e1 conectada.",
-"message.storage.traffic": "Tr\u00e1fego entre os recursos internos do CloudStack, incluindo todos os componentes que se comunicam com o servidor de gerenciamento tais como hosts e m\u00e1quinas virtuais de sistema CloudStack. Por favor, configure o tr\u00e1fego do storage aqui.",
-"message.suspend.project": "Voc\u00ea tem certeza que deseja suspender este projeto ?",
-"message.systems.vms.ready": "VM de Sistema prontas.",
+"message.step.4.desc": "Selecione a rede principal que a sua Inst\u00e2ncia virtual estar\u00e1 conectada.",
+"message.step.license.agreements.continue": "Por favor, respeite todos os acordos de licen\u00e7a para continuar",
```

Comment:
Should be `"message.step.license.agreements.continue": "Por favor, aceite todos os acordos de licen\u00e7a para continuar",`?